tests/e2e/cases/15_uninstall.sh

```
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+# `pvm uninstall <ver>` removes the version dir and pvm ls drops the entry.
+# Driver runs this AFTER fpm has been shut down.
+set -euo pipefail
+source "$(dirname "$0")/../_lib.sh"
+
+step "pvm uninstall $PREVIOUS"
+
+"$PVM_BIN" uninstall "$PREVIOUS"
+
+[[ ! -d "$HOME/.local/share/pvm/versions/$PREVIOUS" ]] \
+    && ok "versions/$PREVIOUS directory removed" \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)